certora/README.md

```diff
@@ -0,0 +1,51 @@
+# Certora 
+
+This folder provides a fully functional skeleton for integrating with Certora.
+There are two configurations:
+
+
+
+## Run From the Command Line
+
+To run the checks from the command line, first install the Python SDK:
+`pip install certora-cli`
+
+Set your Certora Key:  
+`export CERTORAKEY=<personal_access_key>` 
+
+Then run from this directory:  
+`certoraRun ./certora/confs/ghost.conf`  
+ 
+## Files
+
+Folder `specs` contains the specification files: 
+1. `ghost.spec` contains basic rules that are not dependent on any interface.
+1. `moreExmaplesERC20.spec` contains basic examples including ghost, hooks, and multi-contract features.
+
+Folder `confs` contains the configuration files for running the Certora Prover:  
+1. File `ghost.conf` for running `ghost.spec` on `ghost.sol` file.  
+This configuration is suitable for a standalone project, containing:  
+    * external calls are assumed to be non-state changing and returning random value on each call.  
+    * loops are unrolled three times.   
+    * calls to fallbacks are assumed to be side-effect-free and only update the native balance.   
+ 1. File `ghostWithERC20s.conf` running `moreExmaplesERC20.spec` on `ghost.sol` with additional ERC20-related contracts.   
```

Comment:
change the numbering to 2

Reply:
for me md is formatting this numbered, but ok change

Reply:
done